runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -465,7 +465,7 @@ public void reconfigureSequencerServers(Layout originalLayout, Layout newLayout,
      */
     public CompletableFuture<Boolean> asyncSequencerBootstrap(
             @NonNull Layout layout, @NonNull ExecutorService service) {
-
+        log.debug("Bootstrapping with layout: " + layout);
```

Comment:
Could we change this to info?  Could the layout information be useful for bug fixing? If it is already being logged elsewhere, then we can leave it as is.

Reply:
It's important to debug the sequencer bootstrap issues on FD startup, this line tells which layout the sequencer will be boostrapped with, which is usefull